### PR TITLE
Feat/Fix: Add translations support, code refactoring, and server console usage

### PR DESCRIPTION
**Pull request by notkoen**
This PR contains several fixes and feature changes:
1. Minor code refactoring
2. Added proper translation support *(This includes some needed consistency changes in how messages are displayed)*
3. Fixed several commands being unusable from the server console due to the usage of `CPrintToChat`
4. ConVars now use their dedicated methods for retrieving values.

There were several minor changes to the wording of several plugin messages/logs and how they are colored.

### CODE REVIEW

**Comment by maxime1907** [comment]
i thought we already had that logic in multicolors directly no ?

**Comment by Rushaway** [comment]
What is your `MuCo_VERSION` @notkoen ?

**Comment by notkoen** [comment]
Marking this pull request as draft as I will be implementing translation phrases for the non-english speaking players.

**Comment by notkoen** [comment]
> we already have that logic in multicolors with `CReplyToCommand`:
> 
> * https://github.com/srcdslab/sm-plugin-MultiColors/blob/master/addons/sourcemod/scripting/include/multicolors.inc#L445

I will be changing `CPrintToChat` to `CReplyToCommand` to resolve this.

File: addons/sourcemod/scripting/FixMemoryLeak.sp [hunk 1]
```diff
@@ -167,13 +172,10 @@ public Action Hook_OnMapChange(int args)
...
 public Action Hook_OnServerQuit(int args)
 {
-	if (!g_bRestart)
-	{
-		SetupNextRestartCurrentMap();
-		SoftServerRestart();
-		return Plugin_Handled;
-	}
-	return Plugin_Continue;
+	if (g_bRestart) return Plugin_Continue;
```
Comment (maxime1907) [inline addons/sourcemod/scripting/FixMemoryLeak.sp:175]:
```
if (g_bRestart)
  return Plugin_Continue;
```

File: addons/sourcemod/scripting/FixMemoryLeak.sp [hunk 2]
```diff
@@ -284,45 +288,44 @@ stock int GetClientCountEx(bool countBots)
 
 public Action OnRoundEnd(Handle event, const char[] name, bool dontBroadcast)
 {
+	if (!IsRestartNeeded()) return Plugin_Continue;
```
Comment (maxime1907) [inline addons/sourcemod/scripting/FixMemoryLeak.sp:291]: https://github.com/srcdslab/sm-plugin-FixMemoryLeak/pull/12/files#r1773066254

File: addons/sourcemod/translations/FixMemoryLeak.phrases.txt [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        "zho"           "伺服器重啓已被延後！\n(玩家數量一超越插件設定：{1}>{2})"
+        "chi"           "伺服器重启已被延后！\n(玩家数量一超越插件设定：{1}>{2})"
+    }
+
+    "Restart Start Chat"
+    {
+        // 1: Alert
+        "#format"       "{1:t}"
+
+        "en"            "{white}Automatic server restart!.\n{2} {white}Rejoin and have fun!"
```
Comment (Rushaway) [inline addons/sourcemod/translations/FixMemoryLeak.phrases.txt:123]: Your're formating with `{1:t}` but use `{2}`

File: addons/sourcemod/translations/FixMemoryLeak.phrases.txt [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+        "zho"           "伺服器即將重啓！\n記得重連伺服器，祝你玩的愉快!"
+        "chi"           "伺服器即将重启！\n记得重连伺服器，祝你玩的愉快!"
+    }
+
+    "Restart Soon Chat"
+    {
+        // 1: Alert
+        "#format"       "{1:t}"
+
+        "en"            "{default}Automatic server restart at the end of the map.\n{2} {white}Don't forget to rejoin after the restart!"
```
Comment (Rushaway) [inline addons/sourcemod/translations/FixMemoryLeak.phrases.txt:140]: same as https://github.com/srcdslab/sm-plugin-FixMemoryLeak/pull/12/files#r1774541874

**Pull request merged**